main.go

```diff
@@ -87,6 +87,8 @@ var (
 	githubScanIssueComments = githubScan.Flag("issue-comments", "Include issue descriptions and comments in scan.").Bool()
 	githubScanPRComments    = githubScan.Flag("pr-comments", "Include pull request descriptions and comments in scan.").Bool()
 	githubScanGistComments  = githubScan.Flag("gist-comments", "Include gist comments in scan.").Bool()
+	githubScanIgnoreGist    = githubScan.Flag("ignore-gist", "Ignore gist comments in scan.").Bool()
+	githubScanIncludeGist   = githubScan.Flag("include-gist", "Include gist comments in scan.").Bool()
```

Comment:
I think it would be better to have a single flag `--include-gists`, which would fit in with the existing convention of `--include-forks` and `--include-members.

Reply:
Addressed in the following commit